packages/api/src/controllers/asset.ts

```diff
@@ -816,12 +820,13 @@ app.patch(
 
     // update a specific asset
     const { id } = req.params;
-    const asset = await db.asset.get(id);
+    const asset = await db.asset.get( id );
     if (!asset || (asset.userId !== req.user.id && !req.user.admin)) {
       throw new NotFoundError(`asset not found`);
     } else if (asset.status.phase !== "ready") {
       throw new UnprocessableEntityError(`asset is not ready`);
     }
+    validateAssetMeta(meta, asset);
```

Comment:
The return value of this function is being ignored. It is also not the only place where this is called. Anytime you change the behavior of a function you need to update everywhere it is used. So keep that in mind and look for the other usages in the code.

Reply:
Actually... This function is **already** being called in this API handler. You should call it only **once**. No reason to call it again. Remove this line and edit the original call instead.